src/main/java/appeng/parts/reporting/AbstractPartMonitor.java

```diff
@@ -79,6 +79,7 @@
 {
 	private static final IWideReadableNumberConverter NUMBER_CONVERTER = ReadableNumberConverter.INSTANCE;
 	private IAEItemStack configuredItem;
+	private String itemStackSize;
```

Comment:
Yeah, except that this is serverside and thus containing `render` is also not the best idea.